### PR TITLE
Updated list for Open Source Recommender Systems

### PR DESCRIPTION
**Pull request by ExtremelySunnyYK**
- Added a new and rising library for python implicit recommendation
- Discontinued PredictionIO

### CODE REVIEW

**Comment by ExtremelySunnyYK** [comment]
thanks for the contribution. i went through the list and i have made a edit to the current state of open source repo. cheers :)

File: README.md [hunk 1]
```diff
@@ -50,12 +50,13 @@ The open-source recommender systems are:
...
-1. [Gorse](https://gorse.io/) is an offline recommender system backend based on collaborative filtering written in Go. It implements mutiple rated or ranked based recommenders and multiple tools ranging from import/export tools to RESTful recommender server.
-1. [Nvidia Merlin](https://developer.nvidia.com/blog/announcing-nvidia-merlin-application-framework-for-deep-recommender-systems/) is an end-to-end recommender-on-GPU framework that aims to provide fast feature engineering and high training throughput to enable fast experimentation and production retraining of DL recommender models.
+1. [Implicit](https://github.com/benfred/implicit) is a Fast Python Collaborative Filtering for Implicit Datasets. This project provides fast Python implementations of several different popular recommendation algorithms for implicit feedback datasets.
+3. [recommenderlab](https://cran.r-project.org/web/packages/recommenderlab/index.html) provides a research infrastructure to test and develop recommender algorithms including UBCF, IBCF, FunkSVD and association rule-based algorithms.
```
Comment (grahamjenson) [inline README.md:54]:
```suggestion
1. [recommenderlab](https://cran.r-project.org/web/packages/recommenderlab/index.html) provides a research infrastructure to test and develop recommender algorithms including UBCF, IBCF, FunkSVD and association rule-based algorithms.
```

File: README.md [hunk 2]
```diff
@@ -50,12 +50,13 @@ The open-source recommender systems are:
...
+4. [CaseRecommender](https://github.com/caserec/CaseRecommender) is a Python implementation of a number of popular recommendation algorithms. The framework aims to provide a rich set of components from which you can construct a customized recommender system from a set of algorithms.
```
Comment (grahamjenson) [inline README.md:55]:
```suggestion
1. [CaseRecommender](https://github.com/caserec/CaseRecommender) is a Python implementation of a number of popular recommendation algorithms. The framework aims to provide a rich set of components from which you can construct a customized recommender system from a set of algorithms.
```

File: README.md [hunk 3]
```diff
@@ -50,12 +50,13 @@ The open-source recommender systems are:
...
+5. [ProbQA](https://github.com/srogatch/ProbQA) is a C++/CUDA recommender system that uses Bayesian approach to learning how answers to its questions map to best recommendations of a target being searched. On GitHub it's available with an example of learning the binary search algorithm. Its application to a [video game recommendation system](http://probqa.com/) is available on the internet as a demo of the engine.
```
Comment (grahamjenson) [inline README.md:56]:
```suggestion
1. [ProbQA](https://github.com/srogatch/ProbQA) is a C++/CUDA recommender system that uses Bayesian approach to learning how answers to its questions map to best recommendations of a target being searched. On GitHub it's available with an example of learning the binary search algorithm. Its application to a [video game recommendation system](http://probqa.com/) is available on the internet as a demo of the engine.
```

File: README.md [hunk 4]
```diff
@@ -50,12 +50,13 @@ The open-source recommender systems are:
...
+6. [Microsoft Recommenders](https://github.com/Microsoft/Recommenders) contains examples, utilities and best practices for building recommendation systems. Implementations of several state-of-the-art algorithms are provided for self-study and customization in your own applications.
```
Comment (grahamjenson) [inline README.md:57]:
```suggestion
1. [Microsoft Recommenders](https://github.com/Microsoft/Recommenders) contains examples, utilities and best practices for building recommendation systems. Implementations of several state-of-the-art algorithms are provided for self-study and customization in your own applications.
```

File: README.md [hunk 5]
```diff
@@ -50,12 +50,13 @@ The open-source recommender systems are:
...
+7. [Gorse](https://gorse.io/) is an offline recommender system backend based on collaborative filtering written in Go. It implements mutiple rated or ranked based recommenders and multiple tools ranging from import/export tools to RESTful recommender server.
```
Comment (grahamjenson) [inline README.md:58]:
```suggestion
1. [Gorse](https://gorse.io/) is an offline recommender system backend based on collaborative filtering written in Go. It implements mutiple rated or ranked based recommenders and multiple tools ranging from import/export tools to RESTful recommender server.
```

File: README.md [hunk 6]
```diff
@@ -50,12 +50,13 @@ The open-source recommender systems are:
...
+8. [Nvidia Merlin](https://developer.nvidia.com/blog/announcing-nvidia-merlin-application-framework-for-deep-recommender-systems/) is an end-to-end recommender-on-GPU framework that aims to provide fast feature engineering and high training throughput to enable fast experimentation and production retraining of DL recommender models.
```
Comment (grahamjenson) [inline README.md:59]:
```suggestion
1. [Nvidia Merlin](https://developer.nvidia.com/blog/announcing-nvidia-merlin-application-framework-for-deep-recommender-systems/) is an end-to-end recommender-on-GPU framework that aims to provide fast feature engineering and high training throughput to enable fast experimentation and production retraining of DL recommender models.
```

**Comment by grahamjenson** [comment]
This looks great! can you make all list lines start with `1.` markdown will auto give them a number and it makes diffs much smaller.

**Comment by ExtremelySunnyYK** [comment]
> This looks great! can you make all list lines start with `1.` markdown will auto give them a number and it makes diffs much smaller.

okay! i have committed your suggestions

**Pull request merged**